+Base/GitPanel.m

```diff
@@ -0,0 +1,164 @@
+classdef GitPanel
+    
+    properties
+        panel;      % uipanel
+        control;    % uicontrol that displays the current branch info. % For some reason, this property is empty from inside the object, but not from outside. Using panel.Children to access this uicontrol.
+        menu;       % uicontextmenu to show useful git options.
+    end
+    
+    properties (Constant)
```

Comment:
why Constant?

Reply:
'Cause it currently doesn't dynamically update when you set it in runtime. So you shouldn't be able to set it at runtime.

Reply:
But I made it work with runtime.